### ISSUE TITLE
No exit when camera is open

### ISSUE THREAD

**Issue by stacksatsio** [nostr-connect/nostrum#6]
Can’t exit out of the camera when opened to scan a QR code

**Comment by tiero** [comment]
You are right! Contributions are welcome, otherwise need to wait one of the next weekends
